services/cms/index.ts

```diff
@@ -0,0 +1,123 @@
+import { CmsClient, components } from "@cowprotocol/cms"; 
```

Comment:
@alfetopito I hope you are happy now, u made me suffer to leave this one ready :P